src/runtime_src/tools/xclbinutil/SectionAIEPartition.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (C) 2022 Advanced Micro Devices, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef __SectionAIEPartitions_h_
+#define __SectionAIEPartitions_h_
+
+// ----------------------- I N C L U D E S -----------------------------------
+
+// #includes here - please keep these to a bare minimum!
+#include "Section.h"
+#include <boost/functional/factory.hpp>
+
+// -------- C L A S S :   S e c t i o n A I E P a r t i t i o n --------------
+
+class SectionAIEPartition : public Section {
+ public:
+  SectionAIEPartition();
+  virtual ~SectionAIEPartition();
+
+ public:
+  virtual bool doesSupportAddFormatType(FormatType _eFormatType) const;
+  virtual bool doesSupportDumpFormatType(FormatType _eFormatType) const;
```

Comment:
```suggestion
  bool doesSupportAddFormatType(FormatType _eFormatType) const override;
  bool doesSupportDumpFormatType(FormatType _eFormatType) const override;
```
The compiler actually uses override to validate that you indeed are overriding some exact match